sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -282,6 +284,13 @@ def time_to_live(self, value):
         else:
             self._amqp_header.time_to_live = int(value) * 1000
 
+        if self._amqp_header.time_to_live and self._amqp_header.time_to_live != MAX_DURATION_VALUE:
+            self._amqp_properties.creation_time = int(utc_now().timestamp())
+            self._amqp_properties.absolute_expiry_time = min(
+                MAX_ABSOLUTE_EXPIRY_TIME,
```

Comment:
also do other languages default `absolute_expiry_time` to MAX_ABSOLUTE_EXPIRY_TIME?

Reply:
yes --> https://github.com/Azure/azure-sdk-for-js/blob/f53488f738ceb7c76fd1b76f67f36e1c8c868fc0/sdk/servicebus/service-bus/src/serviceBusMessage.ts#L326

Reply:
I see, so this logic only applies when `time_to_live` is set